packages/Babylonian-Core.package/BPExample.class/instance/initializeFromPragma..st

```diff
@@ -3,7 +3,8 @@ initializeFromPragma: aPragma
 	
 	self 
 		method: aPragma method;
-		exampleName: aPragma arguments first.
+		exampleName: aPragma arguments first;
+		expectedResult: aPragma arguments second.
```

Comment:
Is there a specific reason, why the expected result should be at the second position?

Reply:
i want it before the optional parameters for more consistency and thought second place is better than first where the name ist

Reply:
after reading the rest of the review, i will move it to the last position